examples/k8s-patch-basic.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: k8s-patch-basic-
```

Comment:
minor - file name should be `{name}-{kind}.yaml` E.g. k8s-patch-basic-workflow.yaml

Reply:
are you sure? doesn't seem to fit the pattern of the other files in this dir.  I have the change stashed though if you want it, just let me know.  just wanted to make sure :)

Reply:
this is the new way